test/Feature/HLSLLib/ldexp.32.test

```diff
@@ -0,0 +1,62 @@
+#--- source.hlsl
+
+StructuredBuffer<float4> In : register(t0);
```

Comment:
Is there a reason we use float4 instead of float?

Reply:
This way each <n x float> version can easily be tested. If it was float, a float4, etc, would need to be constructed.
Line 8 tests the float4 version of ldexp, like 10 tests float3 etc